packages/wattles/pubspec.yaml

```
@@ -1,5 +1,5 @@
 name: wattles
-description: Wattles is a strongly typed Dart ORM that does not require generating code to function.
+description: Wattle(s) is a strongly typed Dart ORM that does not require generating code to function.
```

```suggestion
description: Wattle(s) is a strongly typed Dart ORM that does not require code generation.
```

nit feel free to ignore.